src/main/java/uk/gov/companieshouse/acsp/controllers/AcspController.java

```diff
@@ -91,15 +84,16 @@ public ResponseEntity<Object> getAcspData(@PathVariable(TRANSACTION_ID_KEY) Stri
         }
     }
 
-    @GetMapping("/acsp-api/user/{acsp_id}/application")
-    public ResponseEntity<Object> checkHasApplication(@PathVariable("acsp_id") String acspId,
-                                                @RequestHeader(value = ERIC_ACCESS_TOKEN) String requestId){
+    @GetMapping("/acsp-api/user/{acsp_application_id}/application")
+    public ResponseEntity<Object> checkHasApplication(@PathVariable("acsp_application_id") String acspId,
+                                                      HttpServletRequest request){
         LOGGER.info("received request to check for user applications");
+        String requestId = request.getHeader(ApiSdkManager.getEricPassthroughTokenHeader());
         return acspService.getAcspApplicationStatus(acspId, requestId);
     }
 
-    @DeleteMapping("/acsp-api/user/{acsp_id}/application")
-    public ResponseEntity<Object> deleteApplication(@PathVariable("acsp_id") String acspId) {
+    @DeleteMapping("/acsp-api/user/{acsp_application_id}/application")
```

Comment:
I don't think we are using this endpoint, If we are not planning to use it in future then we can remove it.